pkg/provisioner/ironic/ironic.go

```diff
@@ -1938,7 +1948,7 @@ func (p *ironicProvisioner) loadBusyHosts() (hosts map[string]struct{}, err erro
 
 	hosts = make(map[string]struct{})
 	pager := nodes.List(p.client, nodes.ListOpts{
-		Fields: []string{"uuid,name,provision_state,driver_internal_info,target_provision_state"},
+		Fields: []string{"uuid,name,provision_state,boot_interface"},
```

Comment:
I don't remember the reason why these fields were requested initially, but if they're not required anymore then it seems fine to do some cleanup

Reply:
I think we wanted to use agent_url from driver_internal_info, but then dropped this idea.

Reply:
Ok. Just wondering if it could be worth moving this refactoring into a separate commit for the sake of clarity with a proper comment (another PR seems overkilling in this case)